packages/fast-components-react-base/src/horizontal-overflow/README.md

```diff
@@ -1,5 +1,8 @@
 # Horizontal overflow
 The *horizontal overflow* component accepts items as children. A child can have an optional *slot* property which is either "next" or "previous" which can be used to allow the next/previous click actions. *Horizontal overflow* uses native scrolling behavior.
 
+### Usage guidance
+The *horizontal overflow* component always initializes at the start of the overflow contianer &mdash; left side in LTR and right side in RTL.
```

Comment:
nit: container, but I would actually change this to content

Reply:
Sure.